electron/launcher/src/_reducers/authentication.reducer.js

```diff
@@ -0,0 +1,49 @@
+const INITIAL_STATE = {
+  user: {
+    username: '',
+    password: '',
+    email: '',
+    birthday: '',
+    isActive: false,
+    isAdmin: false
+  },
+  isAuthenticated: false,
+  token: null,
+  error: null
+}
+
+// Currently not using our payload to make changes to state
+export default (state = INITIAL_STATE, action) => {
+  switch(action.type) {
+    case 'LOGIN_SUCCESS':
+      return { 
+        ...state, 
+        isAuthenticated: true, 
+        user: {
+          ...state.user,
+          username: action.payload.user.username,
+          email: action.payload.user.email,
+          isActive: action.payload.user.isActive,
+          isAdmin: action.payload.user.isAdmin
+        }
+      };
+    case 'LOGIN_FAILED':
+      return { ...state, isAuthenticated: false, error: action.payload.error}
```

Comment:
same this as above - may want to move constant values to a `const` file, and use variable reference instead.

Reply:
Done, reducer now references the constants created in user.actions.type.js instead of hard-coded string.